src/Compilers/Core/Portable/Compilation/Compilation.cs

```diff
@@ -850,6 +850,7 @@ public INamedTypeSymbol GetSpecialType(SpecialType specialType)
         /// <summary>
         /// The TypeSymbol for the type 'dynamic' in this Compilation.
         /// </summary>
+        /// <exception cref="NotSupportedException">If the compilation is a VisualBasic compilation.</exception>
```

Comment:
Was this followup on a previous PR?

Reply:
Yes, [https://github.com/dotnet/roslyn/pull/41970#discussion_r385417667](https://github.com/dotnet/roslyn/pull/41970#discussion_r385417667)

---
In reply to: [389075500](https://github.com/dotnet/roslyn/pull/42143#discussion_r389075500) [](ancestors = 389075500)